### PR TITLE
docs(pt): translate challenge 16

### PR DESCRIPTION
**Pull request by kabrunko-dev**

### CODE REVIEW

File: docs/src/content/docs/pt/challenges/angular/16-di.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+title: 🔴 Dominando Injeção de Dependência
+description: Desafio 16 é sobre dominar como injeção de dependência funciona
+author: thomas-laforge
+contributors:
+  - tomalaforge
+  - tomer953
```
Comment (tomalaforge) [inline docs/src/content/docs/pt/challenges/angular/16-di.md:7]: you need to replave the contributors list with your name.

File: docs/src/content/docs/challenges/angular/16-di.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 title: 🔴 Master Dependancy Injection
-description: Challenge 16 is about masjering how dependancy injection works
+description: Challenge 16 is about mastering how dependancy injection works
 author: thomas-laforge
 contributors:
   - tomalaforge
```
Comment (tomalaforge) [inline docs/src/content/docs/challenges/angular/16-di.md:6]: you can add your name

Comment (kabrunko-dev) [reply docs/src/content/docs/challenges/angular/16-di.md:6]: Done!

File: docs/src/content/docs/pt/challenges/angular/16-di.md (hunk 1 above)
Comment (kabrunko-dev) [reply docs/src/content/docs/pt/challenges/angular/16-di.md]: Done!

**Review by tomalaforge — APPROVED** [review]

**Pull request merged**